src/lib/Scenes/Home/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react"
+import { View } from "react-native"
+import ScrollableTabView from "react-native-scrollable-tab-view"
+import { default as Artists } from "../../Containers/Home"
```

Comment:
Just dropped the current Home view into a tab for now but it'll be replaced with the new Artists tab view

Reply:
:+1: awesome stuff